qiskit_qec/utils/pauli_rep.py

```diff
@@ -2384,7 +2437,12 @@ def symplectic2str(
                 else:
                     tmp_tensor_str = _ENC[output_tensor_encoding][mark] + tmp_tensor_str
             tensor_str.append(tmp_tensor_str)
-    elif syntax == INDEX_SYNTAX:
+    elif syntax in (INDEX_SYNTAX, LATEX_SYNTAX):
+        if syntax == LATEX_SYNTAX:
+            str_repr_index = _ind_to_latex_repr
+        else:
+            str_repr_index = str
+
```

Comment:
Since `str_repr_index` can be index or latex, do we want to rename to something like `str_repr` ?

Reply:
Yes, done

Reply:
Thanks!